staticcheck.conf

```diff
@@ -0,0 +1 @@
+dot_import_whitelist=["github.com/onsi/gomega", "github.com/onsi/gomega/gstruct"]
```

Comment:
I saw that other config files didn't have any header (license), but I can add it if it should be so.

Also I didn't add any comment for the exclusion, but if you prefer to have it, I can comment too

Reply:
Let's add at least.
```
# SPDX-License-Identifier: Apache-2.0
```

Reply:
> Also I didn't add any comment for the exclusion, but if you prefer to have it, I can comment too 

Adding short comment that gomega is heavily used in our tests would not harm but I think it's also not really needed